src/utils/deepLinks.ts

```diff
@@ -19,6 +19,7 @@ export const buildDeepLinkingMap = (orgId: string) => ({
   '/me/nodejsclient': `/orgs/${orgId}/load-data/client-libraries/javascript-node`,
   [`/me/${PROJECT_NAME_PLURAL.toLowerCase()}`]: `/orgs/${orgId}/${PROJECT_NAME_PLURAL.toLowerCase()}`,
   '/me/notebooks': `/orgs/${orgId}/${PROJECT_NAME_PLURAL.toLowerCase()}`,
+  '/me/orglist': `/orgs/${orgId}/accounts/orglist`,
```

Comment:
`orglist` just sits weird with me. I get why it's called that, given we have an `/orgs/$orgid` endpoint, but maybe it should be called `/me/orgs`?

Reply:
I think `/orgs` works, but I have a slight preference for keeping it as something distinct - like `orgslist` - to avoid confusion on whether something is a deep link or not (naming them the same means we'll have a url "/me/orgs" and also "/orgs").

Reply:
yeah, I figured that was what was behind it.